### ISSUE TITLE
PlatformDependent / Low-Level API / Buffers

### ISSUE THREAD

**Issue by tehnewb** [netty/netty#11872]
### Expected behavior
Server bootstrap up and running
### Actual behavior

The server will not boot up and throws this error:

```
Nov 27, 2021 8:29:00 PM io.netty.util.internal.PlatformDependent <clinit>
INFO: Your platform does not provide complete low-level API for accessing direct buffers reliably. Unless explicitly requested, heap buffer will always be preferred to avoid potential system unstability.
Exception in thread "Thread-0" io.netty.channel.ChannelException: Unable to create Channel from class class io.netty.channel.socket.nio.NioSocketChannel
        at io.netty.channel.ReflectiveChannelFactory.newChannel(ReflectiveChannelFactory.java:40)
        at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:319)
        at io.netty.bootstrap.Bootstrap.doResolveAndConnect(Bootstrap.java:163)
        at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:145)
        at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:126)
        at com.game.network.GameNetworkClient.run(GameNetworkClient.java:49)
        at java.base/java.lang.Thread.run(Thread.java:831)
Caused by: java.lang.NoClassDefFoundError: Could not initialize class io.netty.util.internal.PlatformDependent0
        at io.netty.util.internal.PlatformDependent.getClassLoader(PlatformDependent.java:881)
        at io.netty.channel.DefaultChannelId.defaultProcessId(DefaultChannelId.java:110)
        at io.netty.channel.DefaultChannelId.<clinit>(DefaultChannelId.java:78)
        at io.netty.channel.AbstractChannel.newId(AbstractChannel.java:107)
        at io.netty.channel.AbstractChannel.<init>(AbstractChannel.java:79)
        at io.netty.channel.nio.AbstractNioChannel.<init>(AbstractNioChannel.java:84)
        at io.netty.channel.nio.AbstractNioByteChannel.<init>(AbstractNioByteChannel.java:54)
        at io.netty.channel.socket.nio.NioSocketChannel.<init>(NioSocketChannel.java:97)
        at io.netty.channel.socket.nio.NioSocketChannel.<init>(NioSocketChannel.java:87)
        at io.netty.channel.socket.nio.NioSocketChannel.<init>(NioSocketChannel.java:80)
        at io.netty.channel.socket.nio.NioSocketChannel.<init>(NioSocketChannel.java:73)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:78)
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
        at java.base/java.lang.reflect.ReflectAccess.newInstance(ReflectAccess.java:128)
        at java.base/jdk.internal.reflect.ReflectionFactory.newInstance(ReflectionFactory.java:350)
        at java.base/java.lang.Class.newInstance(Class.java:642)
        at io.netty.channel.ReflectiveChannelFactory.newChannel(ReflectiveChannelFactory.java:38)
        ... 6 more
```
### Steps to reproduce
Changed Java versions to 1.7, 1.8, 13, and 16
### Minimal yet complete reproducer code (or URL to code)
```java
public void run() {
	NioEventLoopGroup workerGroup = new NioEventLoopGroup();

	try {
		Bootstrap b = new Bootstrap();
		b.group(workerGroup);
		b.channel(NioSocketChannel.class);
		b.option(ChannelOption.SO_KEEPALIVE, true);
		b.option(ChannelOption.TCP_NODELAY, true);
		b.handler(new ChannelInitializer<Channel>() {
			@Override
			public void initChannel(Channel ch) throws Exception {
				ch.pipeline().addLast("decoder", new GameRawDecoder());
			ch.pipeline().addLast("handler", new GameNetworkHandler());
			}
		});

		ChannelFuture f = b.connect(host, port).sync(); // <---------- THIS CODE THROWS THE ERROR

		f.channel().closeFuture().sync();
	} catch (InterruptedException e) {
		e.printStackTrace();
	} finally {
		workerGroup.shutdownGracefully();
	}
}
```
### Netty version
4.1.7.Final
### JVM version (e.g. `java -version`)
JDK-16.0.2
### OS version (e.g. `uname -a`)
Windows 11

**Comment by hyperxpro** [comment]
Please upgrade to the latest version of Netty and try again.

**Comment by chrisvest** [comment]
You seem to be missing an important internal class. Did you repackage the jar files?

**Comment by tehnewb** [comment]
> Please upgrade to the latest version of Netty and try again.
I am using the latest version Netty has to offer


> You seem to be missing an important internal class. Did you repackage the jar files?

I have re-build the gradle project over and over, so I don't believe it's the case.

**Comment by tehnewb** [comment]
I have found this though that has fixed the issue: netty 4.1.7 is different from netty 4.1.70
